R/redist_smc.R

```diff
@@ -104,21 +116,24 @@
 #' constr = redist_constr(fl_map)
 #' constr = add_constr_incumbency(constr, strength=100, incumbents=c(3, 6, 25))
 #' sampled_constr = redist_smc(fl_map, 10000, constraints=constr)
+#'
+#' # Multiple parallel independent runs
+#' redist_smc(fl_map, 1000, runs=2)
+#'
+#' # One run with multiple cores
+#' redist_smc(fl_map, 1000, cores=2)
 #' }
 #'
 #' @concept simulate
 #' @md
 #' @order 1
 #' @export
 redist_smc = function(map, nsims, counties=NULL, compactness=1, constraints=list(),
-                      resample=TRUE, constraint_fn=function(m) rep(0, ncol(m)),
-                      init_particles=NULL, n_steps=NULL,
-                      adapt_k_thresh=0.985, seq_alpha=0.2+0.3*compactness,
+                      resample=TRUE, runs=1L, cores=0L, init_particles=NULL,
```

Comment:
Can we avoid adding a new term like runs?

Reply:
what would you suggest?

Reply:
Is chains appropriate? If not, then runs makes sense. Just trying to avoid any sort of new glossary situation.

Reply:
yeah in the `plans` object the column is called `chains` but it's definitely not a markov chain so I'm reluctant to adopt it in the actual function.  Dunno if it needs  a glossary entry if it's just for SMC